### ISSUE TITLE
bulk move tries to move previously moved files

### ISSUE THREAD

**Issue by tjex** [nvim-tree/nvim-tree.lua#2040]
### Description

I go about moving a few files. 
The first file move operation with `bmv` (bulk move) works fine. 
The second file move operation follows this sequence: 
1. I get an error that *file 1* does not exist, and cannot be moved (because it already has been)
2. I get a confirmation that *file 2* was moved successfully

If I try to move a third file, the sequence is similar, just one error message longer.
1. I get the same error message about *file 1* not being found
2. I now get an error message that *file 2* cannot be found (as it was moved in the step above)
3. I get a confirmation that *file 3* was moved successfully. 

Here is an asci cinema recording of the process, running with the minimal config

https://asciinema.org/a/AP8cKjqo6ROBLiVQlcaqFr5hv

### Neovim version

```text
NVIM v0.8.3
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by brew@HMBRW-A-001-M1-004.local
```

### Operating system and version

macOS 12.6.1

### nvim-tree version

bbb6d48

### Minimal config

```lua
(no changes made to the nvt-min.lua file)
```

### Steps to reproduce

Steps are clear enough in the asci cinema video above I think. If not, let me know! 🙏

Commands used:
1. mark with `m`
2. move with `bmv`

### Expected behavior

Each file moves successfully without the error message of the previous files not being found anymore. 
Eg nvim-tree not attempting to move files that were previously (and successfully) moved. 

### Actual behavior

https://asciinema.org/a/AP8cKjqo6ROBLiVQlcaqFr5hv

**Comment by alex-courtis** [comment]
Replicated.

**Comment by alex-courtis** [comment]
Fixed #2276

**Comment by tjex** [comment]
legend!
